v1/website/core/Footer.js

```diff
@@ -77,28 +77,28 @@ class Footer extends React.Component {
             <h5>Docs</h5>
             <a
               href={`
-                ${this.props.config.baseUrl}docs/${
+                ${this.props.config.baseUrl}${this.props.config.docsUrl}${
```

Comment:
No more `getDocsUrl` first ?

Reply:
Looks like no need cuz `this.props.config` is passed down from `siteConfig` 😅 Although, might need fallback at `server/routing.js`.